asteroidsyncservice/watch.cpp

```diff
@@ -89,3 +101,14 @@ void Watch::setTime(QDateTime t)
 {
     m_iface->call("SetTime", t.currentDateTime());
 }
+
+bool Watch::timeServiceReady()
+{
+    return m_timeServiceReady;
+}
+
+void Watch::tServiceReady()
+{
+    m_timeServiceReady = true;
```

Comment:
When do you set this value back to false? If you connect to a watch, then disconnect, the value will stay true, right?

Reply:
Do you have a good idea how I can reset the signal?

Reply:
On disconnection